frontend/web/components/LimitAlert.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useState } from 'react'
+import Button from 'components/base/forms/Button'
+
+type LimitAlertType = {
+  limitType: string
+  percentage: number
+}
+
+const LimitAlert: FC<LimitAlertType> = (props) => {
+  const { limitType, percentage } = props
+  return (
+    <>
+      <div
+        className='alert flex-1 align-items-center'
+        style={{ background: '#FDFDDD' }}
```

Comment:
Can we stick to the design system rather than have a static background

Reply:
Added `alert_warning` in the design system.